libavcodec/jpeg2000.c

```diff
@@ -260,17 +260,16 @@ static void init_band_stepsize(AVCodecContext *avctx,
                 band->f_stepsize *= F_LFTG_X * F_LFTG_X * 4;
                 break;
         }
-        if (codsty->transform == FF_DWT97) {
-            band->f_stepsize *= pow(F_LFTG_K, 2*(codsty->nreslevels2decode - reslevelno) + lband - 2);
-        }
+        // scaling
+        band->f_stepsize *= pow(F_LFTG_K, 2*(codsty->nreslevels2decode - reslevelno) + lband - 2);
     }
 
     if (band->f_stepsize > (INT_MAX >> 15)) {
         band->f_stepsize = 0;
         av_log(avctx, AV_LOG_ERROR, "stepsize out of range\n");
     }
 
-    band->i_stepsize = band->f_stepsize * (1 << 15);
+    band->i_stepsize = (int)lround(band->f_stepsize * (1 << 15) + 0.5f);
```

Comment:
```suggestion
    band->i_stepsize = lrint(band->f_stepsize * (1 << 15) + 0.5f);
```

https://github.com/FFmpeg/FFmpeg/blob/d89930f8666c563a96f1dc1835b450522b4cd6da/libavutil/libm.h#L437C43-L437C49

Reply:
Thanks!